src/options/getPackageOptions.ts

```diff
@@ -6,16 +6,34 @@ import { getDefaultOptions } from './getDefaultOptions';
 
 /**
  * Gets all package level options (default + root options + package options + cli options)
+ * This function inherits packageOptions from the rootOptions
  */
-export function getPackageOptions(packagePath: string): PackageOptions {
-  const configExplorer = cosmiconfigSync('beachball', { cache: false });
-  const searchResults = configExplorer.search(packagePath);
+export function getCombinedPackageOptions(actualPackageOptions: PackageOptions): PackageOptions {
   const defaultOptions = getDefaultOptions();
   const rootOptions = getRootOptions();
   return {
     ...defaultOptions,
     ...rootOptions,
-    ...(searchResults && searchResults.config),
+    ...actualPackageOptions,
     ...getCliOptions(),
   };
 }
+
+/**
+ * Gets all the package options from the configuration file of the package itself without inheritance
+ */
+export function getPackageOptions(packagePath: string): PackageOptions {
+  const defaultOptions = getDefaultOptions();
+  const configExplorer = cosmiconfigSync('beachball', { cache: false });
+  try {
+    const results = configExplorer.load(packagePath);
+    return results && results.config;
+  } catch (e) {
+    // File does not exist, returns the default packageOptions
+    return {
+      gitTags: defaultOptions.gitTags,
+      disallowedChangeTypes: defaultOptions.disallowedChangeTypes,
+      defaultNpmTag: defaultOptions.defaultNpmTag,
+    };
```

Comment:
this is not quite right; they will overwrite root config when we combine:
```
    ...rootOptions,
    ...actualPackageOptions,
```

Reply:
i think we should return `Partial<PackageOptions>` type and `{}` value from  `getPackageOption`